src/_kw

```diff
@@ -406,8 +405,7 @@ _kw_man()
   _values 'kw commands' \
     'backup' 'build' 'codestyle' 'config' 'debug' 'deploy' 'device' 'diff' 'drm' 'env' \
     'explore' 'init' 'mail' 'maintainers' 'pomodoro' 'remote' 'report' 'self-update' 'ssh' \
-    'patch-hub' 'vars'
-}
```

Comment:
I think you deleted this curly bracket by mistake 😅. Doing this breaks the Zsh completions, so please put them back.